docs/migration/dfx-0.13.1-migration-guide.md

```diff
@@ -0,0 +1,144 @@
+# 0.13.2 Migration Guide - Environment Variables
```

Comment:
The filename of this migration guide also needs to change

Reply:
@krpeacock Are you wanting this change to make it into dfx 0.14?  This file and its filename would need small changes.